configs/icds-14lakh-dimagivalues-jan2020.yml

```
@@ -0,0 +1,750 @@
+estimation_buffer: 0.24
+estimation_growth_factor: 0.01  # buffer increases by 1% per month
+storage_buffer: 0.33  # keep max storage at 75% of disk
+storage_display_unit: TB
+
+summary_dates:
+#  - '2019-08'
+  - '2020-09'  # 12 months from now
+
+vm_os_storage_gb: 70
+vm_os_storage_group: 'VM_os'
+
+sets_summary_date: '2020-09'
+sets:
+  users:
+    - name: 7lakh
+      users: 700000
+    - name: 8lakh
+      users: 800000
+    - name: 9lakh
+      users: 900000
+    - name: 10lakh
+      users: 1000000
+    - name: 11lakh
+      users: 1100000
+    - name: 12lakh
+      users: 1200000
+    - name: 13lakh
+      users: 1300000
+    - name: 14lakh
+      users: 1400000
+
+  forms_per_user:
+    - name: 400fpu
+      forms_per_user: 400
+
+  vars:
+    - beneficiary_phone_number_seeding: 0.7
```

```suggestion
    - beneficiary_phone_number_seeding: 0.3
```

"Percentage of beneficiary cases that have phone numbers"